core/src/test/scala/org/apache/spark/deploy/rest/SubmitRestProtocolSuite.scala

```diff
@@ -258,6 +260,33 @@ class SubmitRestProtocolSuite extends SparkFunSuite {
       |  }
       |}
     """.stripMargin
+  } else {
+    s"""
+      |{
+      |  "action" : "CreateSubmissionRequest",
+      |  "appArgs" : [ "two slices", "a hint of cinnamon" ],
+      |  "appResource" : "honey-walnut-cherry.jar",
+      |  "clientSparkVersion" : "1.2.3",
+      |  "environmentVariables" : {
+      |    "PATH" : "/dev/null"
+      |  },
+      |  "mainClass" : "org.apache.spark.examples.SparkPie",
+      |  "sparkProperties" : {
+      |    "spark.driver.extraLibraryPath" : "pickle.jar",
+      |    "spark.jars" : "mayonnaise.jar,ketchup.jar",
+      |    "spark.driver.supervise" : "false",
+      |    "spark.driver.memory" : "1024m",
```

Comment:
Maybe `"${Utils.DEFAULT_DRIVER_MEM_MB}m"`?

Reply:
This is intentionally copied from `2.12` test case. If we change `2.13` here, we need to change `2.12`, too.
- https://github.com/apache/spark/pull/29298/files#diff-2ebc20516434bc6dc6ad4e9698e9b15bR239-R261

Reply:
I'd like to avoid touching the existing Scala 2.12 test case~

Reply:
Eh, https://github.com/apache/spark/pull/29298/files#diff-2ebc20516434bc6dc6ad4e9698e9b15bR239-R261 has `"${Utils.DEFAULT_DRIVER_MEM_MB}m"` here

Reply:
Oh, something looks incorrect. Let me check again.

Reply:
Got it. My bad.

Reply:
Thanks! It's fixed!